### DISCUSSION TITLE
[Suggestion] Option to increse ladder hitbox

### DISCUSSION THREAD

**Discussion by XXxinsanityxX** [Roadhog360/Et-Futurum-Requiem#84] [Improvements]
This is to match the hitbox of the ladder to the one of the open trapdoor and doors, It would allow for ladders to be climbed past the trapdoors instead of hitting your head on them.

![HitBox](https://user-images.githubusercontent.com/42390377/137427118-b3c037e5-b358-49db-a329-2bea3cb4cd6c.png)

**Comment by GregoriusT** [comment]
I'm not 100% sure, due to not having MC open, but is this technically a Feature of "Future" Vanilla Minecraft? I know they fixed some things about Trapdoors and Ladders, but how exactly did they do that?

**Comment by XXxinsanityxX** [comment]
Yes this was fixed in newer versions not sure when.

This is 1.16.5 the hitbox of the ladder matches the trapdoor.
![2021-10-14_23 36 55](https://user-images.githubusercontent.com/42390377/137432993-9e0de8d4-4e3e-45ec-9c53-fad7ae2e9339.png)

**Comment by GregoriusT** [comment]
Should be enough Info for when Roadhog is back. ^^

**Comment by Roadhog360** [comment]
Forgot to remove enhancement label, suggestions now belong in the discussions tab.

**Comment by Roadhog360** [comment]
Well, after almost 2 years of waiting (sorry) you finally got it! Fulfilled as of [e2b146f](https://github.com/Roadhog360/Et-Futurum-Requiem/commit/e2b146f1f09bc3c3fea4b62c819a840b40809471).
